### PR TITLE
Restructure `README.md`

### PR DESCRIPTION
**Pull request by qcasey**
I [mentioned](https://github.com/paperless-ngx/paperless-ngx/pull/55#issuecomment-1048045508) it'd be a good idea to restructure the README for clarity. 

These are the changes this PR makes ([View render on GH](https://github.com/paperless-ngx/paperless-ngx/tree/readme-ngx#readme)):

1. Adds Paperless logo to header area (thoughts? too big?).
2. Adds tagline (from docs) to :hook: hook those awesome-self-hosted skimmers.
3. Clarifies the paperless-ngx fork situation.
4. Adds table of contents.
5. Moves **Paperless-ngx vs Paperless** and **Migrating from Paperless to Paperless-ngx** further down.
6. Changes screenshot from dashboard to documents (more compelling IMO).
7. Typo and sentence structures fixes.

Of course, open to suggestions

### CODE REVIEW

**Comment by shamoon** [comment]
Love it. You mind if I suggest some visual sprucing? (yes logo to big imho, more screenshots)

**Comment by qcasey** [comment]
Please do! Remove logo entirely, or compress vertically? Which [screenshots](https://paperless-ngx.readthedocs.io/en/latest/screenshots.html)?

**Comment by shamoon** [comment]
Also... is anyone interested in a new logo?

**Comment by qcasey** [comment]
> Also... is anyone interested in a new logo?

I'd be into it. Would have to pass the community's approval though :smile:

**Comment by tribut** [comment]
While we're at it... I wouldn't mind a name without "ngx" 🙈

**Comment by qcasey** [comment]
> While we're at it... I wouldn't mind a name without "ngx" see_no_evil

That one is difficult because it's important to show at a glance this is a continuation of `paperless-ng`. ngx was chosen hastily though, if there are any other ideas we can field them :)

**Comment by tribut** [comment]
> That one is difficult because it's important to show at a glance this is a continuation of `paperless-ng`.

I know. Maybe a big rebrand for 2.0 😎🤞

**Comment by qcasey** [comment]
Is anyone opposed to adding the demo server we've been using for our android apps?

https://paperless-demo.quinncasey.com
Username: paperless
Password: paperless

It resets @ midnight PT every day

**Comment by shamoon** [comment]
> EDIT: Actually I don't mind the leaf logo itself, the serif text just doesn't fit IMO

Yep, the type =)

The demo is great! Is there any easy way to host it on a less 'personal' domain? Maybe thats overcomplicating it, Im good either way.

**Comment by qcasey** [comment]
I'd also prefer a demo.paperless.org or something similar

**Comment by shamoon** [comment]
Dunno who owns paperless.org https://www.whois.com/whois/paperless.org

Im happy to get a domain (in perpetuity), if that would help but not sure about how to get some kind of free hosting for the app etc

**Comment by qcasey** [comment]
We could get a hip alternative like paperless.xyz for only $89,888.00 USD

Screenshot_20220224-111357_Firefox~2

I'm all for other options. I can leave the demo out for now, just something to consider.

**Comment by mtettke123** [comment]
I coul probably sponsor a VHost and a.e. Paperless-ngx.de if that's ok.

**Comment by shamoon** [comment]
Sorry didn’t see that last comment, I purchased paperless-ngx.com (1y in case we do change they name). when I have a moment I gotta figure out the easiest/cheapest way to host demo

**Comment by qcasey** [comment]
You can point the domain to anyone who'd be willing to host the demo, that'd be free at least.

**Comment by shamoon** [comment]
http://demo.paperless-ngx.com/ (demo / demo) =)

**Comment by tribut** [comment]
How do you feel about removing the whole "Paperless-ngx vs Paperless" section? It made sense when paperless-ng forked, but the Readme is already quite long as-is and I feel the section adds more confusion than anything else.

**Comment by shamoon** [comment]
> How do you feel about removing the whole "Paperless-ngx vs Paperless" section? It made sense when paperless-ng forked, but the Readme is already quite long as-is and I feel the section adds more confusion than anything else.

Lol I just did that
(I agree, obviously =)

**Comment by tribut** [comment]
> Lol I just did that (I agree, obviously =)

Like the change, but I was going to be more blunt. Like this: https://github.com/paperless-ngx/paperless-ngx/pull/148

**Comment by shamoon** [comment]
I figured we could just all commit into this PR for now, easy enough to revert someones commit if needed. I left the migration thing but yea, maybe not needed

**Comment by shamoon** [comment]
Again, not wedded to any of my changes, feel free to undo. I do like the dark mode version of the new screenshot tho, its purdy.

Screen Shot 2022-02-25 at 1 51 08 AM

**Comment by qcasey** [comment]
There are people who are still using danielquinn's paperless, who might want a rundown of the differences. I agree it doesn't belong in the README but should be migrated to the docs.

Great changes!

**Comment by shamoon** [comment]
> There are people who are still using danielquinn's paperless, who might want a rundown of the differences. I agree it doesn't belong in the README but should be migrated to the docs.

Yea great idea, and we just link to it from readme.

**Comment by bauerj** [comment]
Wow, this looks great!

With this prominent macOS dark mode screenshot it sure needs a lot more emojis 😉

File: README.md [hunk 1]
```diff
@@ -66,48 +57,45 @@ If you want to see paperless-ngx in action, [more screenshots are available in t
 	* Paperless learns from your documents and will be able to automatically assign tags, correspondents and types to documents once you've stored a few documents in paperless.
 * Optimized for multi core systems: Paperless-ng consumes multiple documents in parallel.
 * The integrated sanity checker makes sure that your document archive is in good health.
+* [More screenshots are available in the documentation](https://paperless-ngx.readthedocs.io/en/latest/screenshots.html).
 
 # Getting started
 
-The recommended way to deploy paperless is docker-compose. The files in the /docker/compose directory are configured to pull the image from Docker Hub.
+The recommended way to deploy paperless is docker-compose. The files in the `/docker/compose` directory are configured to pull the image from GHCR.io.
```
Comment (bauerj) [inline README.md:64]: There should be a link to the Setup page in the docs here.

Probably also a copy-pastable

```
curl -L https://raw.githubusercontent.com/paperless-ngx/paperless-ngx/master/install-paperless-ng.sh | bash
```

File: README.md [hunk 2]
```diff
@@ -2,50 +2,41 @@
...
 
-2. Wait for paperless to process your files. OCR is expensive, and depending on the power of your machine, this might take a bit of time.
-3. Use the web frontend to sift through the database and find what you want.
-4. Download the PDF you need/want via the web interface and do whatever you like with it.  You can even print it and send it as if it's the original. In most cases, no one will care or notice.
+Paperless-ngx forked from [paperless-ng](https://github.com/jonaswinkler/paperless-ng) to continue the great work and distribute responsibility among a team of people. That discussion can be found in issues
+[#1599](https://github.com/jonaswinkler/paperless-ng/issues/1599)
+and [#1632](https://github.com/jonaswinkler/paperless-ng/issues/1632).
 
-Here's what you get:
+#### Demo
```
Comment (bauerj) [inline README.md:19]: I think this would be fine without the heading

**Review by bauerj — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bauerj) [reply README.md:64]: By the way, is that script currently working?

Comment (qcasey) [reply README.md:64]: After #77, yes I can confirm it's working. If we include that line it should be changed as it was in the PR.

```bash -c "$(curl -L https://raw.githubusercontent.com/paperless-ngx/paperless-ngx/master/install-paperless-ng.sh)"```

File: README.md [hunk 3]
```diff
@@ -2,50 +2,41 @@
...
+A demo is available at [demo.paperless-ngx.com](http://demo.paperless-ngx.com) using login `demo` / `demo` (content is reset frequently).
```
Comment (sytone) [inline README.md:20]: Perhaps update it to be something like this:

A demo is available at [demo.paperless-ngx.com](http://demo.paperless-ngx.com) using login `demo` / `demo` 

> :warning: Warning
> Content is regularly reset and as this is accessible by everyone please do not upload and confidential information.     |

Comment (sytone) [reply README.md:20]: Also add demo to the TOC and move it lower down after features.

File: README.md [hunk 4]
```diff
@@ -66,48 +57,45 @@ If you want to see paperless-ngx in action, [more screenshots are available in t
 	* Paperless learns from your documents and will be able to automatically assign tags, correspondents and types to documents once you've stored a few documents in paperless.
 * Optimized for multi core systems: Paperless-ng consumes multiple documents in parallel.
```
Comment (sytone) [inline README.md:58]: Paperless-ngx

File: README.md (hunk 3 above)
Comment (shamoon) [reply README.md]: I dont think every heading needs to be in TOC (e.g. this one doesnt), and demo is above TOC on purpose. Its often [one of] the first things you see on an OSS readme. Its like, "enough talking just go play with it".

As for warning about the demo, I dont think we want a warning sign so prominent in the readme, especially if the demo stays here. I think the confidential thing is sorta obvious but cant hurt to mention.

I added a commit for these, lets see how others feel.

File: README.md [hunk 5]
```diff
@@ -2,50 +2,40 @@
...
-Paperless-ngx forked from paperless-ng to continue the great work already done and distribute responsibility among a team of people.
-
-Discussion around that can be found in the issues in the paperless-ng repository:
-[1599](https://github.com/jonaswinkler/paperless-ng/issues/1599)
-[1632](https://github.com/jonaswinkler/paperless-ng/issues/1632)
-## Get in Touch
-
-People interested in continuing the work on paperless-ng(x) and form the organisation connected here on github and created a
-[Matrix Room](https://matrix.to/#/#paperless:adnidor.de) for realtime communication.
+![Paperless logo](https://github.com/paperless-ngx/paperless-ngx/raw/main/resources/logo/web/png/Black%20logo%20-%20no%20background.png#gh-light-mode-only)
```
Comment (shamoon) [inline README.md:7]: Does anyone know how we can control the size of the logo but still use the awesome `#gh-dark-mode-only` feature? 

I dont want to do anything too hack-y but most of what I read says have to use an `` tag for sizing, which I think breaks `#gh-dark-mode-only`.

**Comment by shamoon** [comment]
fyi I checked the sizing in that last commit with the new logo, looks good 👍

**Comment by tribut** [comment]
Is it possible to enable https for the demo btw? Browsers are only going to make it harder to open plain http links.

**Comment by shamoon** [comment]
> Is it possible to enable https for the demo? Browsers are only going to make it harder to open plain http links.

Donezo, I was too lazy to wrestle with Traefik yesterday. Gets a little less painful every time 😅

File: README.md (hunk 3 above)
Comment (bauerj) [reply README.md]: IMO the demo should stay where it is now, it's a very important feature.

**Comment by qcasey** [comment]
This is looking really good, thank you all! I cleaned up some of those requested edits. I think we're getting close.

**Review by bauerj — APPROVED** [review]
This is great!

**Comment by sytone** [comment]
Looks great, thanks for the cleanup.

**Review by sytone — APPROVED** [review]

**Pull request merged**